### ISSUE TITLE
Live clients getting out of sync from leader

### ISSUE THREAD

**Issue by Tim020** [dreamteamprod/DigiScript#493]
There are some cases where fast scrolling leads to clients getting out of sync. Improve the handling of client scroll to check that the page/line reference being sent has the page loaded for it, if not it should load the necessary page and buffer pages, jump to that page and then jump to the line in question

**Comment by mstratford** [comment]
Yes. Certainly in live mode I've had it jump 2 pages on from 14 to 16 on my monitor. Maybe there's some edge case with 4:3 monitors or something?
